Windows.Kits.cmake

```diff
@@ -54,8 +54,18 @@ if(NOT CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE)
     set(CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE x64)
 endif()
 
+if(NOT CMAKE_WINDOWS_KITS_10_DIR)
+	set(CMAKE_WINDOWS_KITS_10_DIR "$ENV{WindowsSdkDir}")
```

Comment:
Could you add a comment before the `set`, just to call out that the `WindowsSdkDir` is set by the eWDK?

```cmake
# If the WindowsSdkDir environment variable is set - which the eWDK does - use that for `CMAKE_WINDOWS_KITS_10_DIR`
```

Reply:
I think I may remove this part entirely and require EWDK users to specify CMAKE_WINDOWS_KITS_10_DIR manually. For reasons see my comment on VSINSTALLDIR.